test/prefer-node-append.js

```diff
@@ -0,0 +1,32 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/prefer-node-append';
+
+const ruleTester = avaRuleTester(test, {
+	env: {
+		es6: true
+	}
+});
+
+ruleTester.run('prefer-node-append', rule, {
+	valid: [
+		'parent.append(child);',
+		'document.body.append(child);'
+	],
+	invalid: [
+		{
+			code: 'node.appendChild(child);',
+			output: 'node.append(child);',
+			errors: [{
+				message: 'Prefer `append` over `appendChild`'
+			}]
+		},
+		{
+			code: 'document.body.appendChild(child);',
+			output: 'document.body.append(child);',
+			errors: [{
+				message: 'Prefer `append` over `appendChild`'
+			}]
+		}
```

Comment:
Add tests for `appendChild()` and `appendChild(null)` just to make sure eslint doesn't crash.